### PR TITLE
EPMDJ-5628: Disable settings if agent is offline or unregister

### PR DESCRIPTION
**Pull request by SMPlay**
Issue: EPMDJ-5628

### CODE REVIEW

File: src/pages/agent-full-page/coverage-plugin/coverage-plugin-header/coverage-plugin-header.tsx [hunk 1]
```diff
@@ -83,7 +84,7 @@ export const CoveragePluginHeader = coveragePluginHeader(({ className, previousB
             Build:
             <Version data-test="coverage-plugin-header:build-version">{buildVersion}</Version>
           </CurrentBuild>
-          {baselineInfo && (
+          {(baselineInfo && (agentStatus === AGENT_STATUS.ONLINE || agentStatus === AGENT_STATUS.BUSY)) && (
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/coverage-plugin-header/coverage-plugin-header.tsx:87]: mb only ONLINE status?

File: src/pages/agents-page/agents-table/actions-column/actions-column.tsx [hunk 2]
```diff
@@ -58,7 +60,8 @@ export const ActionsColumn = actionsColumn(({ className, agent }: Props) => {
           height={16}
           width={16}
           data-test="action-column:icons-settings"
-          disabled={status === AGENT_STATUS.NOT_REGISTERED || isOfflineAgent}
+          disabled={(status === AGENT_STATUS.NOT_REGISTERED || status === AGENT_STATUS.OFFLINE)
```
Comment (RomanAchilov) [inline src/pages/agents-page/agents-table/actions-column/actions-column.tsx:63]: !== online

File: src/pages/service-group-full-page/service-group-header/service-group-header.tsx [hunk 3]
```diff
@@ -22,6 +27,12 @@ export const ServiceGroupHeader = serviceGroupHeader(
   }: Props) => {
     const { id = '' } = useParams<{ id: string }>();
     const { push } = useHistory();
+    const agentsList = useWsConnection<Agent[]>(
```
Comment (RomanAchilov) [inline src/pages/service-group-full-page/service-group-header/service-group-header.tsx:30]: very bad decision

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/pages/agent-full-page/coverage-plugin/coverage-plugin-header/coverage-plugin-header.tsx (hunk 1 above)
Comment (SMPlay) [reply src/pages/agent-full-page/coverage-plugin/coverage-plugin-header/coverage-plugin-header.tsx:87]: no
image

File: src/pages/agents-page/agents-table/actions-column/actions-column.tsx (hunk 2 above)
Comment (SMPlay) [reply src/pages/agents-page/agents-table/actions-column/actions-column.tsx:63]: this is doesn't work(
sett

Comment (RomanAchilov) [reply src/pages/agents-page/agents-table/actions-column/actions-column.tsx:63]: status &&

**Review by RomanAchilov — APPROVED** [review]

**Pull request merged**